docs/source/running-autometa.rst

```diff
@@ -2,21 +2,15 @@
 Running Autometa
 ================
 
-Before running anything make sure you
-
-1. Running python3
-2. Have activated the conda environment using:
-
-    ``conda activate autometa``
-3. Are in the Autometa directory
-4. If you are running the main autometa.py make sure that the Autometa directory is in your $PATH environmental variable
+Before running anything make sure you have activated the conda environment using
+ ``conda activate autometa`` and are in the Autometa directory.
```

Comment:
The way it is outputting is a little weird. 
`Before running anything make sure you have activated the conda environment using` is colored in blue and the code is in unusually dark red color.
I guess indentation and a blank line may solve this.

Reply:
Yeah, this is a bit weird, I'll fix the formatting